lib/bundler/plugin/installer.rb

```diff
@@ -38,22 +43,29 @@ def definition.lock(*); end
 
     private
 
+      # Rubocop misunderstands the semantics of this method, assuming an `else` code block
+      # that doesn't exist. See https://github.com/bbatsov/rubocop/issues/5702.
+      #
+      # rubocop:disable Style/GuardClause
+      def check_sources_consistency!(options)
+        if options[:git] && options[:'local-git']
```

Comment:
I believe it would be slightly safer to use `options.has_key? :git && options.has_key? :local_git` here.. someone **shouldn't** do something like `bundle install --git=false` (which would be a string, regardless..) but Just In Case we get a boolean inserted into these further up the chain..

Reply:
Addressed. Note that `:options` key checking was previously done via direct access, so this is a effectively a minor refactoring/improvement, so depending on the project guidelines on refactoring, I could also apply the same to all the other checks in the file - let me know!

Also, the project Rubocop configuration doesn't accept `has_key?` - it requires `key?`, which I've used.